app-backend/db/src/test/scala/com/azavea/rf/database/AnnotationProjectDaoSpec.scala

```diff
@@ -332,4 +335,116 @@ class AnnotationProjectDaoSpec
       )
     }
   }
+
+  test("list projects belonging to a campaign") {
+    check {
+      forAll(
+        (
+            userCreate: User.Create,
+            annotationProjectCreates: List[AnnotationProject.Create],
+            taskFeaturesCreate: Task.TaskFeatureCollectionCreate,
+            campaignCreate: Campaign.Create
+        ) => {
+          val pageSize = 20
+          val pageRequest = PageRequest(0, pageSize, Map.empty)
+
+          val listIO = for {
+            user <- UserDao.create(userCreate)
+            insertedCampaign <- CampaignDao
+              .insertCampaign(campaignCreate, user)
+            insertedProjects <- annotationProjectCreates
+              .take(pageSize) traverse { toInsert =>
+              AnnotationProjectDao
+                .insert(
+                  toInsert.copy(campaignId = Some(insertedCampaign.id)),
+                  user
+                )
+            }
+            _ <- insertedProjects traverse { project =>
+              TaskDao.insertTasks(
+                fixupTaskFeaturesCollection(
+                  taskFeaturesCreate,
+                  project,
+                  Some(TaskStatus.Unlabeled)
+                ),
+                user
+              )
+            }
```

Comment:
Since we don't have any expectations about the tasks here, there's no reason to bother inserting them, and it'll just slow the test down

```suggestion
```